src/layman/layer/filesystem/gdal.py

```diff
@@ -309,12 +309,24 @@ def normalize_raster_file_async(input_path, crs_id, output_file):
     return process
 
 
-def compress_raster_file_async(*, output_file, file_to_compress, ):
+def compress_and_mask_raster_file_async(*, output_file, input_file_path, color_interpretations, nodata_value):
     result_path = output_file
     bash_args = [
         'gdal_translate',
         '-co', 'compress=lzw',
-        file_to_compress,
+    ]
+
+    # GeoServer can visualize grayscale raster with partial opacity from alpha band, but it has unstable contrast.
+    # We don't know how to tell GeoServer to keep partial opacity from alpha band and to stabilize contrast.
+    # We prefer stable contrast over partial opacity, co we create 0/1 mask from alpha band.
```

Comment:
Typo, there is 'c' instead of 's':
"We prefer stable contrast over partial opacity, **s**o we create 0/1 mask from alpha band."